src/main/java/org/c02e/jpgpj/Subkey.java

```diff
@@ -494,8 +492,11 @@ protected PBESecretKeyDecryptor buildDecryptor(char[] passphraseChars) {
         char[] chars = passphraseChars != null &&
             !Arrays.equals(passphraseChars, NO_PASSPHRASE) ?
             passphraseChars : EMPTY_PASSPHRASE;
-        return new BcPBESecretKeyDecryptorBuilder(
-            new BcPGPDigestCalculatorProvider()).build(chars);
+        try {
+            return JCAContextHelper.getJcePBESecretKeyDecryptorBuilder().build(chars);
+        } catch (PGPException e) {
+            throw new RuntimeException(e);
```

Comment:
Instead of wrapping the `PGPException` with a `RuntimeException`, let's just change the signature of this `buildDecryptor` method to `throws PGPException`.